quilt/tools/store.py

```diff
@@ -4,15 +4,24 @@
 import os
 import re
 
+from distutils.dir_util import mkpath
 
 from .const import PACKAGE_DIR_NAME
 from .core import RootNode, CommandException
 from .package import Package
+from .util import BASE_DIR
 
 # start with alpha (_ may clobber attrs), continue with alphanumeric or _
 VALID_NAME_RE = re.compile(r'^[a-zA-Z]\w*$')
 CHUNK_SIZE = 4096
 
+# Helper function to return the default package store path
+def default_store_location():
+    path=os.path.realpath(BASE_DIR)
+    package_dir = os.path.join(path, PACKAGE_DIR_NAME)
```

Comment:
Is `BASE_DIR` actually a good location - as opposed to something more user-friendly or discoverable, like `~/quilt_packages`? Given that it's user data, not settings, the user might want it to be more visible.

Examples I see in my home dir: "~/VirtualBox VMs", "~/Microsoft/TypeScript/node_modules/".

(On the other hand, Steam has an 11GB folder in my "~/.local/share/" dir, so they clearly don't have any problems with this...)

Reply:
I find it pretty convenient myself. The files are named by hashes so it’s not very human-friendly anyway. We could use quilt commands to remove old packages and prune un-connected objects.